VMBackup/main/config/extension.logrotate

```diff
@@ -0,0 +1,23 @@
+/var/log/azure/Microsoft.Azure.RecoveryServices.VMSnapshotLinux/extension.log {
```

Comment:
add handling for extension running with older agent, where log file is created inside extension extension version folder.